examples/state_teleportation.xbb

```diff
@@ -2,30 +2,19 @@
 name StateTeleportation
 # required blackbird API version it targets
 version 1.0
-# optional 'target' specifies the device the blackbird
-# program was compiled for, as well as any other device
-# metadata/options
-target gaussian (shots=1000)
-
-# state to be teleported:
-complex alpha = 1+0.5j
-Coherent(alpha) | 0
 
 # maximally entangled states
-Squeezed(-4) | 1
-Squeezed(4) | 2
+Squeezed(-{sq}) | 1
```

Comment:
how come these were changed?

Reply:
I wanted this to become a parametrized subroutine, so that when called you can alter the value of squeezing (just as an example)